README.md

```diff
@@ -94,6 +94,35 @@ $ mvn install
 
 ## Sending metrics
 
+### Configuring your endpoint
+
+Before we can send metrics to SignalFx, we need to make sure you are sending them to
+the correct SignalFx realm. To determine what realm you are in, check your 
+profile page in the SignalFx web application. Once you know your realm, there
+are three ways to configure the SignalFxReporter class to send to the correct realm:
+
+1. Using the system.properties, add the `com.signalfx.api.port` property with the 
```

Comment:
I think this is not correct.. I think you want com.signalfx.api.hostname=ingest.YOUR_REALM.signalfx.com

Reply:
You are correct. Typo thanks. Will fix.